boards/shields/xenvm_dom0/Kconfig.defconfig

```diff
@@ -6,4 +6,9 @@ if SHIELD_XENVM_DOM0
 config HEAP_MEM_POOL_SIZE
 	default 2097152
 
+# it is needed at least 16M for grant tables
```

Comment:
How about "We need at least 16M of virtual address space to map Xen grant tables"?